### PR TITLE
Adding ipfs-geoip to automated releases

### PR DESCRIPTION
**Pull request by whizzzkid**
Closes: https://github.com/ipfs-shipyard/ipfs-geoip/issues/99

### CODE REVIEW

File: configs/js.json
Comment (whizzzkid) [inline configs/js.json:23]:
```suggestion
    { "target": "ipfs-shipyard/ipfs-geoip" },
    { "target": "ipfs-shipyard/is-ipfs" },
```

File: configs/js.json
Comment (lidel) [inline configs/js.json:22]: I think for this to work, we need `npm-service-account-ipfs` to have write access to https://www.npmjs.com/package/ipfs-geoip  

@protocol/ipdx  I've invited `npm-service-account-ipfs` to that project, are you able to approve NPM invitation? Or is this happening automatically on the first run?

**Review by galargh — APPROVED** [review]

File: configs/js.json
Comment (galargh) [reply configs/js.json:22]: I just noticed, the repo is in `ipfs-shipyard` org, so we should use `npm-service-account-ipfs-shipyard` account here. We will also need to add the NPM token as a secret to the repo. I'm going to do both tomorrow.

Comment (whizzzkid) [reply configs/js.json:22]: @galargh should I wait to merge this or can you please merge this PR when you're good?

Comment (galargh) [reply configs/js.json:22]: I changed the account that has access to the repo from `ipfs` one to the `ipfs-shipyard` one and updated the `NPM_TOKEN` in the repo accordingly.

Comment (galargh) [reply configs/js.json:22]: The release made it to the repo, see https://github.com/ipfs-shipyard/ipfs-geoip/pull/102

Comment (whizzzkid) [reply configs/js.json:22]: I think it'll need some fine-tuning before the tests pass.

**Pull request merged**
